app/src/test/java/com/kickstarter/viewmodels/ThreadViewModelTest.kt

```diff
@@ -27,7 +28,7 @@ class ThreadViewModelTest : KSRobolectricTestCase() {
     private val focusCompose = TestSubscriber<Boolean>()
     private val onReplies = TestSubscriber<List<CommentCardData>>()
 
-    private val createdAt = DateTime.now()
+    // private val createdAt = DateTime.now()
```

Comment:
is there a reason this is commented out? if we don't need it, it should be removed.